From 5993a82d01b67e489429294225c025dcd9725cea Mon Sep 17 00:00:00 2001
From: Kiran Kelageri <kirankelageri@codeaurora.org>
Date: Wed, 22 Aug 2018 12:34:22 -0700
Subject: [PATCH 09/38] Bluetooth-TWS: Disable outgoing connection for 2nd
 earbud.

Change ensures outgoing connection for second earbud on
pairing to disabled by default. if needed can be enabled
by setting property.

 Conflicts:
	packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java

CRs-fixed: 2300140
Change-Id: Id00fdb30825aeee1e6adff55c767e94df96d52b5
Signed-off-by: Volodymyr Zhdanov <wight554@gmail.com>
---
 .../android/settingslib/bluetooth/CachedBluetoothDevice.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java b/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java
index fc54b5e4..6ef9a3a0 100644
--- a/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java
+++ b/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java
@@ -737,7 +737,7 @@ public class CachedBluetoothDevice implements Comparable<CachedBluetoothDevice>
         refresh();
 
         if (bondState == BluetoothDevice.BOND_BONDED) {
-            if (SystemProperties.getBoolean("persist.vendor.btstack.connect.peer_earbud", true)) {
+            if (SystemProperties.getBoolean("persist.vendor.btstack.connect.peer_earbud", false)) {
                 Log.d(TAG, "Initiating connection to" + mDevice);
                 if (mDevice.isBondingInitiatedLocally() || mDevice.isTwsPlusDevice()) {
                     connect();
-- 
2.28.0

